api/Clients.json

```diff
@@ -0,0 +1,302 @@
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "get": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Clients/get",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "51"
+            },
+            "chrome_android": {
+              "version_added": "51"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "45",
+              "notes": "Service workers (and Push) have been disabled in the Firefox 45 & 52 Extended Support Releases (ESR)."
+            },
+            "firefox_android": {
+              "version_added": "45"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "38"
+            },
+            "opera_android": {
+              "version_added": "38"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "matchAll": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Clients/matchAll",
+          "support": {
+            "webview_android": [
+              {
+                "version_added": "40"
+              },
+              {
+                "version_added": "46",
+                "notes": "<code>Client</code> objects returned in most recent focus order."
+              }
+            ],
+            "chrome": [
+              {
+                "version_added": "40"
+              },
+              {
+                "version_added": "46",
+                "notes": "<code>Client</code> objects returned in most recent focus order."
+              }
+            ],
+            "chrome_android": {
+              "version_added": "40"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "44",
+                "notes": "Service workers (and Push) have been disabled in the Firefox 45 & 52 Extended Support Releases (ESR)."
+              },
+              {
+                "version_added": "45",
+                "notes": "<code>includeUncontrolled</code> support."
```

Comment:
The `includeUnControlled` option should be a sub feature of `matchAll`. (much like you added a subfeature `async_waitUntil ` to `waitUntil` in https://github.com/mdn/browser-compat-data/pull/1006#discussion_r167990620).

```
"includeUncontrolled_option": {
  "__compat": {
  "description": "<code>includeUncontrolled</code> option",
```
This way, a separate row will appear for this in the new table.

Thank you!